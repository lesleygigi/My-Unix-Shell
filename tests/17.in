./tests/decho.sh 1 delayed &
./tests/decho.sh 2 delayedmore &
echo now
wait
echo later
