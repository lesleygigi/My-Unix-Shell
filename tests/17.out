./tests/decho.sh 1 delayed &
./tests/decho.sh 2 delayedmore &
echo now
now
wait
delayed
delayedmore
echo later
later
